universite-du-quebec-a-montreal.csl

```diff
@@ -679,6 +702,11 @@
   <macro name="issued-year">
     <choose>
       <if variable="issued">
+        <choose>
+          <if type="book" match="all" variable="translator">
```

Comment:
why are you testing for translator here? That seems odd for original date -- that's frequently for historic works (and you're not testing for translator below)

Reply:
Our style guide distinguishes between translated and untranslated books and it should not in my humble opinion. The guide will be reviewed about this. For now, I'm going to remove the "translator" condition.